### PR TITLE
Bluespace Event NFZ announcements

### PR DESCRIPTION
**Pull request by blackknight954**
## About the PR
I added an additional line of text to the Syndicate Cache and NT Vault announcements stating that there is a three hundred meter NFZ around said entities, and failing to comply would result in persecution

## Why / Balance
There are *very frequent debates* regarding the eligabilities of NFZ's around events, but these are two that are basically universally agreed that they should have NFZ's around them, This would hopefully cause people to stop having a jab at NFSD over these events having NFZ's
## How to test

Load up a test server, 

trigger the event

Witness the new text that was added.

## Media
Changes to Cache announcement
image
Changes to Vault announcement
image

## Requirements
- [x] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [x] I have added media to this PR or it does not require an ingame showcase.

## Breaking changes
There should not be any.

**Changelog**
- Tweak: Updated the NT Vault and Syndicate Cache spawn announcements to include a line about an NFZ being in effect, and to stay clear of the structure.

### CODE REVIEW

**Comment by Houtblokje** [comment]
Will this mean that the syndicate unidentified will no longer be valid to call an NFZ on?

**Comment by Leander-0** [comment]
I think the correct word would be prosecution, persecution is a whole different can of worms, about the "three hundred meters" i would say just make it "300m" to make the message a little smaller and help the slow readers.

besides that i think either the SOP or the space law should also reflect this changes for the bluespace events too but i can do those changes later.

**Comment by blackknight954** [comment]
> I think the correct word would be prosecution, persecution is a whole different can of worms, about the "three hundred meters" i would say just make it "300m" to make the message a little smaller and help the slow readers.
> 
> besides that i think either the SOP or the space law should also reflect this changes for the bluespace events too but i can do those changes later.

Made those changes, today I learned the difference between persecute and prosecute.

**Review by whatston3 — COMMENTED** [review]
I've made suggestions to shorten the text given, as well as the ending text, for both of the events.

Honestly, the guidance coming down from Centcomm makes sense (doing something? too bad, this is valuable, you're on guard duty now).  I'm not really fond of the range being a fixed amount in the text itself rather than a "guard this, figure out how, but feel free to do XYZ" prompt.  Wouldn't "A No Fly Zone may be declared." do the job?

File: Resources/Locale/en-US/_NF/bluespace-events/events.ftl
Comment (blackknight954) [reply Resources/Locale/en-US/_NF/bluespace-events/events.ftl:2]: >Wouldn't "A No Fly Zone may be declared." do the job?

It would, but that falls down to the NFSD to perform, and at least in my experiences it leads to frequent arguing about it.
I'm fine with it either way though.

Comment (blackknight954) [reply Resources/Locale/en-US/_NF/bluespace-events/events.ftl]: The range is too much, my line of thinking was by having the NFZ’s announced by the game announcement rather than by a player would stop any arguments about people getting angry at an SR or sherif for declaring an NFZ, or angry at them for not declaring an NFZ, in the case of other NFSD members.

Comment (blackknight954) [reply Resources/Locale/en-US/_NF/bluespace-events/events.ftl]: @Leander-0 had made an idea thread on the discord regarding it, can ping you later.

File: Resources/Locale/en-US/_NF/bluespace-events/events.ftl
Comment (whatston3) [inline Resources/Locale/en-US/_NF/bluespace-events/events.ftl:1]:
```suggestion
station-event-bluespace-vault-start-announcement = A NanoTrasen armored vault is inbound to the sector. The NFSD may declare the vault a No Fly Zone. Station accounts will be rewarded for its safe return.
```

File: Resources/Locale/en-US/_NF/bluespace-events/events.ftl
Comment (whatston3) [inline Resources/Locale/en-US/_NF/bluespace-events/events.ftl:4]:
```suggestion
station-event-bluespace-cache-start-announcement = A Syndicate weapons cache is inbound to the sector. The NFSD may declare the cache a No Fly Zone. Station accounts will be rewarded for guarding the cache.
```

**Review by whatston3 — APPROVED** [review]
Seems fair to me, I've left suggestions if the extent of the NFZ should be left to the NFSD (in my opinion, should be, a smaller zone is easier to patrol, might depend on the size of the force), and there should be somebody to enforce it now, or they let their slot close, so there's no worry about the event saying an NFZ is active without anyone to enforce it.

Leaving this open pending more discussion.

File: Resources/Locale/en-US/_NF/bluespace-events/events.ftl
Comment (blackknight954) [reply Resources/Locale/en-US/_NF/bluespace-events/events.ftl:1]: With Dvir's change requiring that a Sheriff is present for these events to spawn, I feel like having a static number is fine. Sheriff bein on usually means a increase in NFSD

**Comment by Houtblokje** [comment]
I think this would clear up a lot of confusion around NFZ, so I think this would be a great change. 
The only thing I see that could be a problem is the "may declare", which implies that it is optional (it is, but its also a really valid use of the NFZ) which could lead to arguing that it is unnecessary. Something like "CentCom recommends that NFSD declares it an NFZ" might be better, since at that point it basically becomes an order from CentCom.

**Comment by Jakumba** [comment]
Broadly positive change imo! SOP and Space Law currently conflict on the matter of NFZs and it leads to messy, frustrating RP for a lot of people. This is an excellent change to provide some clarity.
300m might be a bit too large, especially when they can spawn near POIs like trade, but this could be tweaked later.

**Comment by whatston3** [comment]
> 300m might be a bit too large, especially when they can spawn near POIs like trade, but this could be tweaked later.

Imagine if it was left to the sheriff to make the call.  Madness.

**Comment by Leander-0** [comment]
As much as i would love for it to be just a sheriff call and be over with clearly sheriffs feel doubtful on when to apply it, and if they not apply it at all some would feel like they are not doing their job correctly and the ones that might apply too many NFZ then would be believe they are too strict and hoarding the events only for the NFSD.

Sadly both cases just give bad light on to whoever is commanding the NFSD at that moment so this would make things way more simple for them and everyone involved, sheriff will still be able to make NFSD to other kinds of issues if needed.

**Comment by whatston3** [comment]
> As much as i would love for it to be just a sheriff call and be over with clearly sheriffs feel doubtful on when to apply it, and if they not apply it at all some would feel like they are not doing their job correctly and the ones that might apply too many NFZ then would be believe they are too strict and hoarding the events only for the NFSD.
> 
> Sadly both cases just give bad light on to whoever is commanding the NFSD at that moment so this would make things way more simple for them and everyone involved, sheriff will still be able to make NFSD to other kinds of issues if needed.

"An NFZ may be enforced" gives you _explicit permission_ to enforce this, and tacitly gives the responsibility over how large it is to the person carrying the hammer.  A fixed range may be too large for the actual force you have available, which may not be in the interests of the force.

I am not suggesting that things stay the same, I'm asking for wiggle room for the person whose responsibility it is to look after this thing.  If there's a maximum range for an NFZ and concerns about the maximum number that can be reasonably enforced at a time, again, that's fair, but it's an SOP issue.

File: Resources/Locale/en-US/_NF/bluespace-events/events.ftl
Comment (whatston3) [reply Resources/Locale/en-US/_NF/bluespace-events/events.ftl:1]: I am still of the opinion that this should be their call - do not make a call for them, _that is their job_.

Comment (Cheackraze) [reply Resources/Locale/en-US/_NF/bluespace-events/events.ftl:1]: i think I agree with whatston3 here, let the sheriff decide. If they are occupied with a pirate chase or simply dont have the numbers to respond, or any other reasons, they could choose not to guard the cache etc.

File: Resources/Locale/en-US/_NF/bluespace-events/events.ftl
Comment (whatston3) [reply Resources/Locale/en-US/_NF/bluespace-events/events.ftl:10]: Merging these suggestions - if it turns out a fixed range makes sense in play, happy to revert.

**Review by whatston3 — APPROVED** [review]
Merged in the suggestions for the vault reword and the "NFSD may declare" statement rather than a fixed distance.  Good to go on my end, and if this doesn't work out in the wash, can revert it without issue.

**Pull request merged**
